graph/model/flair_template.go

```diff
@@ -0,0 +1,40 @@
+package model
+
+import "time"
+
+type FlairTemplate struct {
+	ID          string     `json:"id" dynamodbav:"ID" gorm:"type:varchar(36);primary_key;"`
+	DisplayName *string    `json:"displayName" gorm:"type:varchar(128);"`
+	ImageURL    *string    `json:"imageURL" gorm:"type:text;"`
+	Source      string     `json:"source" gorm:"type:varchar(128);not null;"`
+	CreatedAt   time.Time  `json:"createdAt" gorm:"not null;default:CURRENT_TIMESTAMP;"`
+	UpdatedAt   time.Time  `json:"updatedAt" gorm:"not null;default:CURRENT_TIMESTAMP ONUPDATE CURRENT_TIMESTAMP;"`
+	DeletedAt   *time.Time `json:"deletedAt"`
+
+	// Flairs      []Flair    `json:"flairs" gorm:"foreignkey:TemplateID;"`
```

Comment:
nit: Should we delete this?

Reply:
✅ https://github.com/nedrocks/delphisbe/pull/2/commits/2a19543827f9fdca1f9387af04e756b7b824f3bf#diff-5d0a34340a502af9b29f2212cf6bda83L14